UI/src/assetTypeConfiguration.js

```diff
@@ -285,19 +301,23 @@
           var datePeriodField = _.filter(fields, function(field) { return field.getPropertyValue().propertyType === 'date_period'; });
 
           var isInDatePeriod = function(date) {
```

Comment:
Same as before:
```suggestion
			var isInDatePeriod = function(date) {
				_.reduce(datePeriodValueExtract(date), function(startDate, endDate){ 
					return new Date(endDate.getMonth() + '/' + endDate.getDate() + '/' + (endDate.getFullYear() - 1)) <= startDate;
				});
			};
```

Check first.

Reply:
I choose Miguel solution! 👍